content/home/examples/a-component-using-external-plugins.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: A Component Using External Plugins
+title: బాహ్య ప్లగిన్లు ఉపయోగించి ఒక భాగం
 order: 3
 domid: markdown-example
 ---
 
-React allows you to interface with other libraries and frameworks. This example uses **remarkable**, an external Markdown library, to convert the `<textarea>`'s value in real time.
+ప్రతిస్పందన మిమ్మల్ని ఇతర గ్రంథాలయాలు మరియు ఫ్రేమ్వర్క్లతో ఇంటర్ఫేస్కు అనుమతిస్తుంది. ఈ ఉదాహరణ **remarkable**, బాహ్య మార్క్డౌన్ లైబ్రరీని ఉపయోగిస్తుంది, నిజ సమయంలో `<textarea>` s విలువను మార్చడానికి.
```

Comment:
change ```ప్రతిస్పందన``` to ```రియాక్ట్ ``` = it is the actual library name so don't literally translate! 

Interface - > ```సమన్వయము కల్గించు``` 

maybe use this sentence

```రియాక్ట్ ఇతర ఫ్రేంవర్క్ తో మరియు లైబ్రరీలతొ సమన్వయము కల్గించును ``` 

second part of the sentence 

change మార్కడను to మార్క్ డౌన్

Reply:
remove s after text area